featurehub-sdk/featurehub-sdk.gemspec

```diff
@@ -34,8 +34,8 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "concurrent-ruby", "~> 1.1.10"
-  spec.add_dependency "faraday", "~> 2.3"
+  spec.add_dependency "concurrent-ruby", "~> 1.1"
+  spec.add_dependency "faraday", "~> 1"
```

Comment:
this is actually a major version change

Reply:
For now I didnt encounter any issues with faraday being 1 with this sdk.
And sadly in my case It can only work with faraday being 1 even after loosing up all other dependancies in my gemfile.
You may leave this pull request open for a while so incase I faced an issue with my branch I will update accordingly.

Reply:
Hello @rvowles, I haven't encountered any issues with my changes until now. From my side it seems save to merge.

Reply:
Heya!

Yeah, however it would be a breaking change for me to drop the version - while the SDK does support Faraday 1.x it can't support 1.x and 2.x and I would prefer to support the newer version of it as we already have large organisations using it... 

What I will be doing is documenting that people can clone and use 1.x themselves if they wish as the SDK does work. Appreciate you coming back and letting us know :-)